src/cider/nrepl/middleware/debug.clj

```diff
@@ -464,6 +464,7 @@
       ;; We do not go so far as to actually try to read the code of the function
       ;; at this point, which is at macroexpansion time.
       (and v
+           (not= 'clojure.core (ns-name (:ns m)))
```

Comment:
Even if we hardcode this temporarily, it probably makes sense to check for all built-in namespaces (or at least all the commonly used ones).